src/main/java/net/sf/jabref/model/event/EntryChangedEvent.java

```diff
@@ -1,6 +1,5 @@
 package net.sf.jabref.model.event;
```

Comment:
This probably needs to be discussed, but is a two minute operation once decided. I can see advantages with both approaches.